lib/engines/hckinstall/hckinstall.rb

```diff
@@ -32,17 +32,12 @@ def self.tag(options)
       options.install.platform
     end
 
-    def read_platform
-      platform_name = @project.options.install.platform
+    def self.platform(logger, options)
+      platform_name = options.install.platform
       platform_json = "#{PLATFORMS_JSON_DIR}/#{platform_name}.json"
 
       @logger.info("Loading platform: #{platform_name}")
-      unless File.exist?(platform_json)
-        @logger.fatal("#{platform_name} does not exist")
-        raise(InvalidConfigFile, "#{platform_name} does not exist")
-      end
-
-      Json.read_json(platform_json, @logger)
+      Json.read_json(platform_json, logger)
     end
```

Comment:
looks like it should be in commit  `Do not check file existence before calling Json.read_json`

Reply:
This method has two changes. The file existence check gets removed first, and then it gets converted into a class method. This line is changed due to the latter conversion.